<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hcmgis_medialaxis_form</class>
 <widget class="QDialog" name="hcmgis_medialaxis_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>231</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Skeleton/Medial Axis</string>
  </property>
  <widget class="QDialogButtonBox" name="BtnOKCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>197</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LblInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Polygon (1..100 selected features limit!)</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="CboInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblInput_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Density (m)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="CboField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblInput_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unique field (when selected features &gt; 1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>167</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>(Notice: Output should be refined after running)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>hcmgis_medialaxis_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>hcmgis_medialaxis_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
